runtime/src/block_cost_limits.rs

```diff
@@ -40,6 +41,7 @@ lazy_static! {
         (solana_vote_program::id(), COMPUTE_UNIT_TO_US_RATIO * 70),
         // secp256k1 is executed in banking stage, it should cost similar to sigverify
```

Comment:
this comment needs to be updated to mention `ed25519`

Reply:
On second thought, might as well to remove this comment - it adds more confusion if anything.